build.sbt

```diff
@@ -30,6 +30,9 @@ lazy val anorm = project
   .settings(
   scalacOptions += "-Xlog-free-terms",
     binaryIssueFilters ++= Seq(
+      missMeth("anorm.DefaultParameterValue.stringValue"/* deprecated */),
+      missMeth("anorm.ParameterValue.stringValue"/* deprecated */),
+      missMeth("anorm.BatchSql.apply"/* was deprecated */),
```

Comment:
Okay, but that means this will require a 2.6 (or greater) version of anorm, right? What is anorm's policy about removing methods?